src/azul/service/manifest_service.py

```diff
@@ -90,7 +90,12 @@ def get_manifest(self, format_: str, filters: Filters) -> Tuple[str, bool]:
         manifest was reused and False if a new manifest was generated.
         """
         generator = ManifestGenerator.for_format(format_, self, filters)
-        object_key, presigned_url = self._get_cached_manifest(generator, format_, filters)
+
+        # Passing 'object_key' as a parameter to `get_manifest()` indicates a
```

Comment:
Rethink the wording. It is imprecise. Also, this needs to go into the docstring.